e2e/setup/evm/setup.go

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2022 LimeChain Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package evm
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/limechain/hedera-eth-bridge-validator/app/clients/evm"
+	"github.com/limechain/hedera-eth-bridge-validator/app/clients/evm/contracts/router"
+	"github.com/limechain/hedera-eth-bridge-validator/app/clients/evm/contracts/wtoken"
+	"github.com/limechain/hedera-eth-bridge-validator/app/domain/client"
+	"github.com/limechain/hedera-eth-bridge-validator/app/domain/service"
+	"github.com/limechain/hedera-eth-bridge-validator/app/model/asset"
+	evm_signer "github.com/limechain/hedera-eth-bridge-validator/app/services/signer/evm"
+)
+
+type Utils struct {
+	EVMClient               *evm.Client
+	EVMFungibleTokenClients map[string]client.EvmFungibleToken
+	EVMNftClients           map[string]client.EvmNft
+	RouterContract          *router.Router
+	KeyTransactor           *bind.TransactOpts
+	Signer                  *evm_signer.Signer
+	Receiver                common.Address
+	RouterAddress           common.Address
+	WTokenContractAddress   string
+}
+
+func RouterAndEVMTokenClientsFromEVMUtils(evmUtils map[uint64]Utils) (
```

Comment:
Looks like this function isn't used anywhere. I think we can remove it or we can use it but to remove it from `e2e/setup/config.go`?